app/views/admin/budget_phases/_form.html.erb

```diff
@@ -43,6 +43,20 @@
     </div>
   <% end %>
 
+  <% if feature?(:allow_images) %>
+    <div class="images small-12 column">
+      <%= render "images/nested_image", imageable: @phase, f: f %>
+      <p class="help-text"><%= t("admin.budget_phases.edit.image_description") %></p>
+    </div>
+  <% end %>
+
+  <div class="small-12 medium-6 column">
+    <p class="form-label"><%= t("admin.budget_phases.edit.main_call_to_action") %></p>
+    <p class="help-text"> <%= t("admin.budget_phases.edit.main_call_to_action_description") %></p>
+    <%= f.text_field :main_link_text %>
```

Comment:
The phase `main_link_text` is not translatable the same as the budget `main_link_text` field. We should make it translatable and move the form inputs to translatable fields sections. I'm working on this.